mutex_windows.go

```diff
@@ -95,25 +196,54 @@ func releaseSemaphone(handle syscall.Handle) error {
 	return nil
 }
 
-func waitForSingleObject(handle syscall.Handle) error {
-	noWait := 0
-	result, _, errno := syscall.Syscall(procWaitForSingleObject.Addr(), 2, uintptr(handle), uintptr(noWait), 0)
+func waitForMultipleObjects(handles []syscall.Handle, timeout time.Duration) (int, error) {
```

Comment:
I'm trusting your wrapper for this the ranging with _WAIT_ABANDONED_0 seems really strange.

Do you have a link for the docs so that we can have someone else understand this in the future?

Reply:
Added some comments and a link to the WaitForMultipleObjects API docs.